c/parallel/test/test_for.cpp

```diff
@@ -151,8 +152,9 @@ struct large_state_t
   int* d_counter;
   int y, z, a;
 };
-extern "C" __device__ void op(large_state_t* state, int* a) {
-  atomicAdd(state->d_counter, *a);
+extern "C" __device__ void op(void* state_ptr, void* a_ptr) {
+  large_state_t* state = static_cast<large_state_t*>(state_ptr);
+  atomicAdd(state->d_counter, *static_cast<int*>(a_ptr));
```

Comment:
Ditto here. Perhaps changes to use `void *` in definition of the operation here are not necessary.